src/PrestaShopBundle/Resources/config/services/bundle/serializer.yml

```diff
@@ -1,17 +0,0 @@
-services:
-  _defaults:
-    public: true
```

Comment:
Mistake or intentional ?

Reply:
Intentional. I've deleted the serializer for same reason of case conventions (which is suppose to be camelCase). This one was for serializing objects to snake_case and its not used anymore. ( as it was introduced also in one of create order page migration prs. recently)